docs/source/user_guide/workflows.rst

```diff
@@ -0,0 +1,492 @@
+=====================================
+LSMO calc functions and work chains
+=====================================
+
+In the following section all the calc functions and work chains of the `aiida-lsmo` plugin are listed and documented.
+
+Force Field Builder
++++++++++++++++++++++++
+
+The FFBuilder allows to combine the force field parameters (typically for a Lennard-Jones potential) for a
```

Comment:
one nice thing you can do in sphinx is to directly link to the source code using 
```suggestion
The :py:func:`~aiida_lsmo.calcfunctions.ff_builder` allows to combine the force field parameters (typically for a Lennard-Jones potential) for a
```

Reply:
just a minor comment looking at the FFBuilder code - probably, this is a case for a class rather than loose functions (but it's not urgent).
In that case, your calcfunction simply becomes a call to the FFBuilder constructor (`ffb = FFBuilder(...)`) & returns something like `ffbuilder.out_dict`.

Reply:
I used the links and I will consider transforming the FFBuilder into a Class